BraveShared/BraveStrings.swift

```diff
@@ -395,6 +395,43 @@ extension Strings {
     }
 }
 
+// MARK: - OptionsMenu
+
+extension Strings {
+    public struct PrivacyFeature {
+        public static let menuSectionTitle = NSLocalizedString(
+            "optionsMenu.menuSectionTitle",
+            tableName: "BraveShared",
+            bundle: Bundle.braveShared,
```

Comment:
all these entries can be simplified by typing `.braveShared` only

Reply:
at https://github.com/brave/brave-ios/pull/4424/commits/47a8bdda827ab7bdcc0013e09f6b346c0202a9a6